src/job.h

```diff
@@ -29,9 +29,9 @@ class Job {
 public:
   Job() = default;
 
-  inline const auto &GetSubGraphs() const { return subgraphs_; }
-  inline const auto &GetId() const { return id_; }
-  inline const auto &GetSubmissionTime() const { return submission_time_; }
+  inline auto &GetSubGraphs() { return subgraphs_; }
```

Comment:
Why not return the const reference? You are not supposed to change the member variable by Get interface. BTW, you don't need to return reference for fundamental type.